web/templates/_base.html

```diff
@@ -3,10 +3,12 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Prometheus Time Series Collection and Processing Server</title>
-    <script src="/static/vendor/js/jquery.min.js"></script>
+    <script src="{{ pathPrefix }}static/vendor/js/jquery.min.js"></script>
 
-    <link type="text/css" rel="stylesheet" href="/static/vendor/bootstrap-3.3.1/css/bootstrap.min.css">
-    <link type="text/css" rel="stylesheet" href="/static/css/prometheus.css">
+    <link type="text/css" rel="stylesheet" href="{{ pathPrefix }}static/vendor/bootstrap-3.3.1/css/bootstrap.min.css">
+    <link type="text/css" rel="stylesheet" href="{{ pathPrefix }}static/css/prometheus.css">
+
+    <script>var PATH_PREFIX = "{{ pathPrefix }}";</script>
```

Comment:
This line will also be needed in `console_libraries/prom.lib`, as consoles use that as their base template. (tested and it works when adding it there)